components/charts-skeletons.js

```diff
@@ -0,0 +1,19 @@
+export function GrowthPieChartSkeleton ({ height = '250px', minWidth = '200px' }) {
+  return <ChartSkeleton {...{ height, minWidth }} />
+}
+
+export function WhenComposedChartSkeleton ({ height = '300px', minWidth = '300px' }) {
+  return <ChartSkeleton {...{ height, minWidth }} />
+}
+
+export function WhenAreaChartSkeleton ({ height = '300px', minWidth = '300px' }) {
+  return <ChartSkeleton {...{ height, minWidth }} />
+}
+
+export function WhenLineChartSkeleton ({ height = '300px', minWidth = '300px' }) {
+  return <ChartSkeleton {...{ height, minWidth }} />
+}
+
+function ChartSkeleton (props) {
+  return <div className='mx-auto w-100 clouds' style={{ ...props }} />
+}
```

Comment:
I went back and forth on the approach here, trying to keep a dedicated skeleton component for each chart component while reducing duplication, and also keeping things configurable. I don't love it, but this seems a decent middle ground...